src/sortStudents.ts

```diff
@@ -1,16 +1,55 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[]
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+const getAver = (marks:number[]): number => {
+  const averageMark:number = marks
+    .reduce((prev: number, curr: number): number => prev + curr, 0)
+    / marks.length;
```

Comment:
You can return immediately
You don't need typify callback , you are working with  array that is already typed
```suggestion
  return grades
    .reduce((acc, curr):  => acc + curr, 0)
    / marks.length;
```